chapters/hpmor-chapter-015.tex

```diff
@@ -62,15 +62,15 @@ \chapter{Conscientiousness}
 Professor McGonagall stood up and moved over to the wall behind her desk, which held a polished wooden board. “There are many reasons why Transfiguration is dangerous, but one reason stands above all the rest.” She produced a short quill with a thick end and used it to sketch letters in red, which she then underlined, using the same marker, in blue:
 
 \begin{center}
-\newsavebox{\hpbox}%
-\fontspec[ExternalLocation,Color=AA0000]{Florante}
-\savebox{\hpbox}{\MakeUppercase{Transfiguration is not permanent!}}
-\vspace{0.5ex}
-\usebox{\hpbox}
-\settowidth{\versewidth}{\usebox{\hpbox}}
-\vskip -1ex
-\fontspec[ExternalLocation,Color=2020FF]{ArchitectsDaughter}
-\resizebox{\versewidth}{.6ex}{\rotatebox{90}{I}}
+  \newsavebox{\hpbox}%
```

Comment:
This change of indentation looks like it is meant for a different PR.

Reply:
sorry, forgot to reply, now as desired in master and no change of that file in this PR